### ISSUE TITLE
Realtime-Chat-Application

### ISSUE THREAD

**Issue by subhangi2731** [Vishal-raj-1/Awesome-JavaScript-Projects#1417]
Real-time chat uses web-based apps that allow communication in a multi-user environment that is usually addressed directly but is anonymous among users. 
![image](https://user-images.githubusercontent.com/53938490/118096940-aadb8180-b3ef-11eb-9f7b-7992ed3738ce.png)

**Comment by subhangi2731** [comment]
@ridsuteri @Vishal-raj-1 please assign this issue to me

**Comment by subhangi2731** [comment]
@ridsuteri please assign me

**Comment by subhangi2731** [comment]
please assign me this issue @ridsuteri @991rajat @harshgupta20

**Comment by Ankita297** [comment]
@subhangi2731  any updates ?

**Comment by Jyoti6828** [comment]
@Vishal-raj-1 Can you please assign this issue to me. I want to work on it.

**Comment by Ankita297** [comment]
@Jyoti6828 go for it .

**Comment by Vishal-raj-1** [comment]
@Jyoti6828 any update ?

**Comment by Jyoti6828** [comment]
@Vishal-raj-1 It's almost completed. I will upload it tomorrow.

**Comment by Ankita297** [comment]
@Jyoti6828 any updaters ?

**Comment by Jyoti6828** [comment]
@Ankita297  Yes.. I created a PR a day ago , but then yesterday from @Rizwan-S I got to know that this repo is allows only HTML , CSS and vanilla JavaScript. Then I asked him that I can build this using required technologies. Now I'm working on it.

**Comment by Ankita297** [comment]
Okay Sorry @Jyoti6828 my bad . I have not seen that . You can go ahead .
